lib/system/base/network.toit

```diff
@@ -67,6 +118,21 @@ abstract class ProxyingNetworkServiceDefinition extends ServiceDefinition:
   */
   abstract proxy_mask -> int
 
+  /**
+  Opens the proxied network.
+
+  Subclasses may decide to use the call to establish the underlying
+    connection in which case the call may throw exceptions and
+    possibly time out. In case of such exceptions, $close_network
+    is not called.
```

Comment:
Does this mean it is someone's responsibility to call close_network?  If so whose?